ASTree.cpp

```diff
@@ -157,6 +157,7 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
         }
 
         switch (opcode) {
+        case Pyc::BINARY_OP_A:
```

Comment:
The `_A` implies that there is an operand; in this case, the operand is the binary operation to perform (it's not just for addition).  You'll need to add a whole separate case for this.